### ISSUE TITLE
Exceptions condition by browser_mod deviceId

### ISSUE THREAD

**Issue by hwmland** [maykar/custom-header#97]
Would be nice to be able to use deviceId (or even better - alias) as condition filter. Something like:

```
custom_header:
  compact_mode: true
  exceptions:
    - conditions:
        device: my_android_phone, 99980b13-dabc9563
      config:
        hide_help: true
        menu_dropdown: true
```

**Comment by DavidFW1960** [comment]
I agree. I have the exact same user_agent in safari on my ipad as on my mac which means I can't use the filter as is now.

**Comment by maykar** [comment]
This will be in the next release, but only as device ID. I'm not sure that alias is something that I could even access, but if it is I would probably have to add card-tools as a requirement for Custom Header and I'd like to avoid that.

**Comment by DavidFW1960** [comment]
I think device ID works in my case.

**Comment by maykar** [comment]
This has been added or fixed in the [latest version 1.2.2](https://community.home-assistant.io/t/custom-header/155399/367). After updating be sure to check your browser's dev console (usually F12) to make sure you are running the latest version. If it doesn't display 1.2.2 you may either need to clear your cache or reinstall.

**Comment by hwmland** [comment]
Thank you, works great both on PC (Win10+Chrome) as well as on Android.
